test/usecases/nonspecific_current/leonhard.mod

```diff
@@ -17,5 +17,17 @@ PARAMETER {
 }
 
 BREAKPOINT {
+        func()
+        func_with_v(v)
+        func_with_other(c)
         il = c * (v - 1.5)
 }
+
+PROCEDURE func() {
+}
+
+PROCEDURE func_with_v(v) {
+}
+
+PROCEDURE func_with_other(q) {
+}
```

Comment:
This isn't related to NONSPECIFIC_CURRENT. We have a other usecases about functions/procedures `func_proc` (+pnt).

Maybe reuse one of those or start a new MOD file. Also please make it a bit stronger, be actually checking functionality. Currently, it only checks that everything compiles.

Reply:
`nrn_cur` seems to be generated only when NONSPECIFIC_CURRENT is present, so I figured why not add the test directly there.

Regarding the functionality checking, I'm not sure how exactly this should be tested as the only thing missing are the variables, which prevent compilation from succeeding. I guess your concern is that something is not being passed correctly (by value instead of reference, etc.)?

Reply:
The empty function bodies feel a lot like something a bug can work its way around quite easily. Then there's the aspect that it runs a simulation and asserts values. But those values are not affected in any way be the code under test.

If you only want to check that it compiles, you could replace `simulate.py` with:
```
# This usecase only checks that the generated code compiles.
print("success: ...")
```

Reply:
Noted, `simulate.py` is now just that.